pkg/option/config.go

```diff
@@ -218,6 +218,9 @@ const (
 	// K8sSyncTimeout is the timeout since last event was received to synchronize all resources with k8s.
 	K8sSyncTimeoutName = "k8s-sync-timeout"
 
+	// HubbleL7ParserHttpRemoveUrlQuery
+	HubbleL7ParserHttpRemoveUrlQuery = "hubble-l7-parser-http-remove-url-query"
```

Comment:
This constant is a left-over of a previous version of the patch correct?

Reply:
Yeap, sorry for these 😞